malloc-trace/src/orphan_container.cc

```diff
@@ -0,0 +1,304 @@
+/**
+ * Copyright 2024 Centreon
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ */
+
+#include <fmt/format.h>
+#include <boost/stacktrace.hpp>
+
+#include "orphan_container.hh"
+
+using namespace com::centreon::malloc_trace;
+
+/**
+ * @brief Construct a new backtrace info::backtrace info object
+ *
+ * @param allocated address allocated or freed
+ * @param allocated_size size allocated
+ * @param thread_id
+ * @param funct_name
```

Comment:
I was hoping to find part of my answer here. I'm still puzzled between malloc, free and information.